src/ceph-disk/ceph_disk/main.py

```diff
@@ -74,6 +84,14 @@
             'ready': 'cafecafe-9b03-4f30-b4c6-35865ceff106',
             'tobe': '89c57f98-2fe5-4dc0-89c1-35865ceff2be',
         },
+        'block.db': {
+            'ready': '166418da-c469-4022-adf4-b30afd37f176',
+            'tobe': '7521c784-4626-4260-bc8d-ba77a0f5ff97',
```

Comment:
s/d28/2be/ because it's cute and convenient to quickly figure out what is tobe (2be)... :-)